src/VisualStudio/Core/Def/Implementation/TaskList/ExternalErrorDiagnosticUpdateSource.cs

```diff
@@ -84,16 +82,15 @@ public void ClearErrors(ProjectId projectId)
             // capture state if it exists
             var state = BuildInprogressState;
 
-            var asyncToken = _listener.BeginAsyncOperation("ClearErrors");
-            _taskQueue.ScheduleTask(() =>
+            _taskQueue.ScheduleTask("ClearErrors", () =>
```

Comment:
```suggestion
            _taskQueue.ScheduleTask(nameof(ClearErrors), () =>
```

(and possibly the same for the other strings in this pr)?

Reply:
Should ScheduleTask take `[CallerMemberName]` attribute then, rather then manually passing the name it?

Reply:
i would be fine withthat.

Reply:
That doesn't quite work that well. I'm gonna keep the string and use `nameof`